Sources/Foundation/URL.swift

```diff
@@ -149,7 +149,10 @@ public struct URLResourceValues {
     
     /// The date the resource was created.
     public var creationDate: Date? {
-        get { return _get(.creationDateKey) }
+        get {
+          guard let timestamp: Int = _get(.creationDateKey) else { return nil }
+          return Date(timeIntervalSince1970: TimeInterval(timestamp))
+        }
```

Comment:
Wait, `set` doesn't have the corresponding issue?

Reply:
Thats actually why Im not 100% certain why this is needed.  There are other `Date` accessors which should also see similar things?  The test suite seems to be passing though, so Im not sure what is special here.